packages/react-instantsearch-hooks-dom/src/widgets/__tests__/RefinementList.test.tsx

```diff
@@ -0,0 +1,3462 @@
+                class="ais-RefinementList-item"
+              >
+                <label
+                  class="ais-RefinementList-label"
+                >
+                  <input
+                    class="ais-RefinementList-checkbox"
+                    type="checkbox"
+                    value="Sony"
+                  />
+                  <span
+                    class="ais-RefinementList-labelText"
+                  >
+                    Sony
+                  </span>
+                  <span
+                    class="ais-RefinementList-count"
+                  >
+                    350
+                  </span>
+                </label>
+              </li>
+              <li
+                class="ais-RefinementList-item"
+              >
+                <label
+                  class="ais-RefinementList-label"
+                >
+                  <input
+                    class="ais-RefinementList-checkbox"
+                    type="checkbox"
+                    value="Incipio"
+                  />
+                  <span
+                    class="ais-RefinementList-labelText"
+                  >
+                    Incipio
+                  </span>
+                  <span
+                    class="ais-RefinementList-count"
+                  >
+                    320
+                  </span>
+                </label>
+              </li>
+              <li
+                class="ais-RefinementList-item"
+              >
+                <label
+                  class="ais-RefinementList-label"
+                >
+                  <input
+                    class="ais-RefinementList-checkbox"
+                    type="checkbox"
+                    value="KitchenAid"
+                  />
+                  <span
+                    class="ais-RefinementList-labelText"
+                  >
+                    KitchenAid
+                  </span>
+                  <span
+                    class="ais-RefinementList-count"
+                  >
+                    318
+                  </span>
+                </label>
+              </li>
+              <li
+                class="ais-RefinementList-item"
+              >
+                <label
+                  class="ais-RefinementList-label"
+                >
+                  <input
+                    class="ais-RefinementList-checkbox"
+                    type="checkbox"
+                    value="Whirlpool"
+                  />
+                  <span
+                    class="ais-RefinementList-labelText"
+                  >
+                    Whirlpool
+                  </span>
+                  <span
+                    class="ais-RefinementList-count"
+                  >
+                    298
+                  </span>
+                </label>
+              </li>
+            </ul>
+          </div>
+        </div>
+      `);
+
+      userEvent.type(searchInput, 'app');
+
+      await wait(0);
+
```

Comment:
We can use `toHaveBeenCalledTimes()` here.

Reply:
Addressed in [445448c](https://github.com/algolia/react-instantsearch/pull/3381/commits/445448cb0cadfd537e1e71a6f7074db1e9789439).